python/py-pip/Portfile

```diff
@@ -59,12 +59,18 @@ if {${name} ne ${subport}} {
         xinstall -m 755 -d ${destroot}${prefix}/share/doc/${subport}/reference
         xinstall -m 644 {*}[glob -types f ${worksrcpath}/docs/reference/*] ${destroot}${prefix}/share/doc/${subport}/reference
 
-        delete ${destroot}${prefix}/bin/pip[string range ${python.version} 0 end-1]-${python.branch}
-        delete ${destroot}${prefix}/bin/pip${python.branch}-${python.branch}
+        delete ${destroot}${prefix}/bin/pip[string range ${python.language_version} 0 0]-${python.branch}
+        delete ${destroot}${prefix}/bin/pip${python.language_version}-${python.branch}
     }
 
     select.group        ${python.rootname}
-    select.file         ${filespath}/${python.rootname}${python.version}
+
+    # use different naming conventions to avoid ugliness like `pippy3` :(
```

Comment:
This also seems inconvenient.

Reply:
I'll check if this applies to other ports, and if so, extract the logic to the PortGroup. Is that better?